From a4567483c403daae7d4d0a295371d23f1db0eb6c Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 17 Jan 2023 11:36:09 +0530
Subject: [PATCH] libgbm is not used by IA.

Intel uses its own minigbm and virtrenderer so this is
not needed.

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 Android.bp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 1b3a2e81..de8c06af 100644
--- a/Android.bp
+++ b/Android.bp
@@ -126,7 +126,6 @@ cc_library {
             shared_libs: [
                 "libdrm",
                 "libepoxy",
-                "libgbm",
             ],
         },
         linux_glibc: {
-- 
2.17.1

